### DISCUSSION TITLE
Open with (VS)Codium on Windows?

### DISCUSSION THREAD

**Discussion by ghost** [VSCodium/vscodium#1315] [Q&A] [answered]
How does one add an entry for (VS)Codium in Windows' context menu? Like VSCode's "Open with code"? I have installed through `winget` and ".msi", I'm not given any options to do this and I did not find any discussion about this (maybe I missed some).

I'm using Codium x64-1.72.2.22289

**Comment by matt-goldman** [comment]
Also want to know how to do this. Based on some of the issues I've seen raised (specifically [this](https://github.com/VSCodium/vscodium/issues/250) and [this](https://github.com/VSCodium/vscodium/issues/369)) it seems like this is functionality that exists. But I don't see it and don't know how to enable it.

**Reply by JacksonBailey** [reply]
See [this](https://github.com/VSCodium/vscodium/discussions/1315#discussioncomment-6480974) if you're using the msi (or want to switch to it from winget) as I had the same problem. It's just hidden under the advanced installation options and off by default.

**Comment by JacksonBailey** [comment]
I installed via the msi and did not see it either. I just reinstalled to check. The problem is that those options are hidden under advanced options and off by default. When you're agreeing to the terms instead of clicking Install click Advanced and you should be good. You may need to uninstall first, I didn't test very much. Regardless, I now see them.

I know your question is about winget and not the msi but this is where the search engine took me and I had the same problem.

**Comment by sneakylenny** [comment, marked as answer]
If you used the MSI you can run the setup again, click "change" (top option) and activate the options there. After that it's just a matter of clicking "change" and you're done.
